index.js

```diff
@@ -25,7 +25,8 @@ module.exports = render;
 
 function render(view, opts) {
   opts = opts || {};
-  var ext = opts.engine || extname(view).slice(1);
+  var _extname = extname(view).slice(1);
+  var ext = opts.engine || _extname === 'jade' ? 'pug' : _extname;
```

Comment:
If we need convert `jade` to `pug` automatically, it should be consolidate's logic.